01_login.php

```diff
@@ -133,107 +166,61 @@
 
 							<div class="w3-row">
 								<div class="w3-col m6 w3-center w3-padding-large">
-									<img src="images/download_low.png" class="w3-round w3-image" alt="logo brand" width="350" height="233">
+									<img src="images/download_low.png" class="w3-mobile w3-round w3-image" alt="logo brand" width="310" height="213" style="margin-top: 15px;margin-right: -50px;">
```

Comment:
w3-mobile made logo responsive with the style sheet positioning the code + change size to fit the screen nicely